modules/sdk-coin-bsc/test/unit/transactionBuilder/send.ts

```diff
@@ -0,0 +1,168 @@
+import { getBuilder } from '../getBuilder';
+import { Ecdsa, ECDSA, TransactionType } from '@bitgo/sdk-core';
+
+describe('BSC Transfer Builder', () => {
+  describe('TSS Signing with 2 of 3', () => {
+    let MPC: Ecdsa;
+    let A_combine: ECDSA.KeyCombined, B_combine: ECDSA.KeyCombined, C_combine: ECDSA.KeyCombined;
+    let serializedTransaction: Buffer;
+
+    before('key generation and unsigned tx building', async () => {
+      MPC = new Ecdsa();
+
+      const A = await MPC.keyShare(1, 2, 3);
```

Comment:
this keyshare creation can be expensive and creates flaky test.  can you use the values from fixtures folder ?

Reply:
Yeah, I did notice flaky tests in the ci yesterday, thanks for pointing out a fix. Just pushed the changes.

Reply:
there's already fixtures keyshares, please use that modules/bitgo/test/v2/fixtures/tss/ecdsaFixtures.ts

Reply:
Left unchanged as discussed (due to no export from bitgo)